File: is52038b-labs/07/java/ListVisualiserDriver.java
public class ListVisualiserDriver {
    public static void main(String args[]) {
        SLList a = new SLList(1, new SLList(11, new SLList(23, SLList.NIL)));
	ListVisualiser v = new ListVisualiser(a);
        v.visualise();
    }
}

File: is52038b-labs/07/cpp/ListVisualiserDriver.cpp
#include "SLList.hpp"
#include "ListVisualiser.hpp"

int main() {
  SLList *a = new SLList(1, new SLList(11, new SLList(23, SLList::NIL)));
  ListVisualiser v = ListVisualiser(a);
  v.visualise();
  return 0;
}
